src/content/CloudBoilerplateNet/Services/ReactiveCacheManager.cs

```diff
@@ -93,14 +95,17 @@ public async Task<T> GetOrCreateAsync<T>(IEnumerable<string> identifierTokens, F
                     // If it doesn't exist, get it via valueFactory.
                     T response = await valueFactory();
 
-                    // Create it in a background thread.
-                    if (createCacheEntriesInBackground)
+                    if (productionValidator(response))
```

Comment:
I'd rename `productionValidator` to something like `skipCacheDelegate`. (And adjust/revert the actual conditions accordingly.)